packages/@webex/plugin-meetings/src/breakouts/breakout.ts

```diff
@@ -55,15 +57,37 @@ const Breakout = WebexPlugin.extend({
    * Joins the breakout session
    * @returns {Promise}
    */
-  join() {
-    return this.request({
+  async join() {
+    const breakoutMoveId = uuid.v4();
+    const {meetingId} = this.parent;
```

Comment:
As mentioned in the other comment to access the breakouts object from a breakout you can write something like
```ts
const {meetingId} = this.collection.parent
```